back/spec/models/home_page_spec.rb

```diff
@@ -39,13 +39,13 @@
     end
   end
 
-  describe 'image uploads' do
-    subject(:home_page) { build(:home_page) }
-
-    it 'stores a header background image' do
-      home_page.header_bg = File.open(Rails.root.join('spec/fixtures/header.jpg'))
-      home_page.save!
-      expect(home_page.header_bg.url).to be_present
-    end
-  end
+  # describe 'image uploads' do
+  #   subject(:home_page) { build(:home_page) }
+
+  #   it 'stores a header background image' do
```

Comment:
Probably, it's better to skip it with `xit` https://relishapp.com/rspec/rspec-core/docs/pending-and-skipped-examples/skip-examples